### PR TITLE
Remove sign up page and phase 3 from testnet landing page

### PR DESCRIPTION
**Pull request by ygao76**
## Summary
Screen Shot 2023-03-17 at 7 10 02 PM
Screen Shot 2023-03-20 at 12 05 28 PM

Screen Shot 2023-03-17 at 4 45 09 PM

## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
```

### CODE REVIEW

File: pages/about.tsx [hunk 1]
```diff
@@ -95,57 +81,6 @@ export default function About({ showNotification, loginContext }: AboutProps) {
           buttonLink={!loaded ? '/signup' : ''}
           disableButton={true}
         />
-        <div className={clsx('mx-6', 'px-3', 'w-full', 'lg:w-2/3', 'mb-6')}>
```
Comment (danield9tqh) [inline pages/about.tsx:98]: Do we want to delete all these instructions? I feel like this is most of the FAQ for this page

Comment (ygao76) [reply pages/about.tsx:98]: This is Phase 3 testnet guideline. Its on the phase3.testnet.ironfish.network.
I dont think we need the phase 3 tesnet info on the testnet website.

File: apiClient/client.ts [hunk 2]
```diff
@@ -212,8 +211,23 @@ export async function login(email: string): Promise<any> {
...
+        case RPCErrorCode.MagicLinkRateLimited:
+          return new LocalError('MagicLinkRateLimited', UNABLE_TO_LOGIN)
+        case RPCErrorCode.UserAlreadyLoggedIn:
+          return new LocalError('UserAlreadyLoggedIn', UNABLE_TO_LOGIN)
+        default:
+          break
+      }
+    }
+
+    return new LocalError((err as Error).message, UNABLE_TO_LOGIN)
```
Comment (NullSoldier) [inline apiClient/client.ts:230]: Is this related to removing the signup page? Should this be another PR?

File: pages/about.tsx [hunk 3]
```diff
@@ -95,57 +81,6 @@ export default function About({ showNotification, loginContext }: AboutProps) {
...
-            >
-              {callsToAction.columnTwo.map(renderColumn)}
-            </div>
-            <div
-              className={clsx('text-center', 'flex', 'md:hidden', 'mx-auto')}
-            >
-              <BasicLink href="#guidelines">View Testnet Guidelines</BasicLink>
-            </div>
-          </div>
-        </div>
```
Comment (NullSoldier) [inline pages/about.tsx:148]: Is this related to removing signup? This looks like it's removing phase 3 stuff from the page, which is good.

File: apiClient/client.ts (hunk 2 above)
Comment (ygao76) [reply apiClient/client.ts:230]: This PR remove sign up and phase 3 from testnet landing page.

**Comment by skylarrichard123** [comment]
Looks like we lost all the padding underneath the login button.

**Comment by skylarrichard123** [comment]
Screen Shot 2023-03-20 at 9 07 05 AM
Add 64px margin bottom (button)

**Review by skylarrichard123 — CHANGES_REQUESTED** [review]
(see above)

**Comment by NullSoldier** [comment]
If skylar approves this, I think it's fine from my side.

**Comment by skylarrichard123** [comment]
> If skylar approves this, I think it's fine from my side.

I've requested a change and then it should be good

**Comment by ygao76** [comment]
@skylarrichard123 added padding

**Review by skylarrichard123 — APPROVED** [review]
